templates/default/sssd.conf.erb

```diff
@@ -25,19 +23,8 @@ realmd_tags = manages-system joined-with-samba
 cache_credentials = True
 krb5_store_password_if_offline = True
 default_shell = /bin/bash
+ldap_sasl_authid = <%= @sasl_authid %>$
```

Comment:
We're using GSSAPI to bind to the directory now? Just curious: why is only the computer name being used here vs "host/myhost" as shown in the docs? Is this some sort of workaround for the hostname issues we've been having?